ee/tabby-db/src/integrations.rs

```diff
@@ -12,8 +13,8 @@ pub struct IntegrationDAO {
     pub display_name: String,
     pub access_token: String,
     pub api_base: Option<String>,
-    pub created_at: DateTimeUtc,
-    pub updated_at: DateTimeUtc,
+    pub created_at: NaiveDateTime,
```

Comment:
should use `DateTime<UTC>`. Applied to other DAOs' as well

Reply:
But the macro `query!` only support field of `NaiveDateTime`. Otherwise we must implement `From<NaiveDateTime>` for other types

Reply:
I check all code and I found that we did not read this field directly. Instead, we construct `Integration` and read it. So I think using `DateTime<Utc>` in `Integration` is enough.

Reply:
It's documented in the deprecation notes:

https://demo.tabbyml.com/files/github/TabbyML/tabby/-/blob/48d9c084cc80d20bd439102cc0768920f6680e91/ee/tabby-db/src/lib.rs?plain=1#L299

Example:
1. https://demo.tabbyml.com/files/github/TabbyML/tabby/-/blob/48d9c084cc80d20bd439102cc0768920f6680e91/
ee/tabby-db/src/threads.rs?plain=1#L78
2. https://demo.tabbyml.com/files/github/TabbyML/tabby/-/blob/48d9c084cc80d20bd439102cc0768920f6680e91/ee/tabby-db/src/threads.rs?plain=1#L189